src/cincinnati/mod.rs

```diff
@@ -236,7 +236,13 @@ fn find_update(
         .find(|(_, node)| is_same_checksum(node, &booted_depl))
     {
         Some(current) => current,
-        None => return Ok(None),
+        None => {
+            log::warn!(
```

Comment:
Fine here, though this is something we'll probably have to undo as part of https://github.com/coreos/fedora-coreos-tracker/issues/1872.